pkg/controller/servicecatalog/provisionedproduct/utils.go

```diff
@@ -10,21 +11,22 @@ import (
 	"k8s.io/utils/pointer"
 
 	svcapitypes "github.com/crossplane-contrib/provider-aws/apis/servicecatalog/v1alpha1"
-	aws "github.com/crossplane-contrib/provider-aws/pkg/clients"
 )
 
-func provisioningParamsAreChanged(cfStackParams []cfsdkv2types.Parameter, currentParams []*svcapitypes.ProvisioningParameter) bool {
-	if len(cfStackParams) != len(currentParams) {
```

Comment:
This check will be produce false positive isUpToDate check If we okay with that it is not necessary to specify every parameter

Reply:
But if so the problem idk how to detect removed parameters